ci-operator/step-registry/gcp/deprovision/vpc/gcp-deprovision-vpc-commands.sh

```diff
@@ -4,9 +4,6 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-python3 --version 
```

Comment:
I guess we do not need to remove the lines for safer guard.

Reply:
Hmm, I meant no need to remove the lines including the following line 8.

Reply:
Reverted my change, thanks!